### DISCUSSION TITLE
Crash on PasPi OS Bullseye with PulseAudio

### DISCUSSION THREAD

**Discussion by Narsskrarc** [dtcooper/raspotify#515] [Q&A] [answered]
Hi there,

I try to get raspotify running on an RasPi 3 with Raspberry Pi OS 11 alias Bullseye and an hifiberry dac clone.
The RasPi is running headless. It's running an own app which uses sound already. Since the OS uses pulseaudio now, I hoped I could use raspotify in parallel, but each time I try to connect I get
`Feb  9 09:04:59 toshio librespot[1071]: Failed to get home directory.`
`Feb  9 09:04:59 toshio librespot[1071]: [2022-02-09T08:04:59Z ERROR librespot_playback::player] Audio Sink Error Connection Refused: <PulseAudioSink> Connection refused`
`Feb  9 09:04:59 toshio systemd[1]: raspotify.service: Main process exited, code=exited, status=1/FAILURE`
`Feb  9 09:04:59 toshio systemd[1]: raspotify.service: Failed with result 'exit-code'.`
`Feb  9 09:04:59 toshio systemd[1]: raspotify.service: Consumed 1.393s CPU time.`
`Feb  9 09:05:09 toshio systemd[1]: raspotify.service: Scheduled restart job, restart counter is at 3.`
`Feb  9 09:05:09 toshio systemd[1]: Stopped Raspotify (Spotify Connect Client).`
`Feb  9 09:05:09 toshio systemd[1]: raspotify.service: Consumed 1.393s CPU time.`
`Feb  9 09:05:09 toshio systemd[1]: Started Raspotify (Spotify Connect Client).`

Librespot has a problem. I tried to google it, but didn't find anything helpful.
I even couldn't find out if the pulsaudio server is running as a system service; my app works even if I'm not logged in (via ssh), but I didn't find the mechanism that starts pulseaudio (there is no pulseaudio.service).

So - what's the problem? Configuration? Problem with pulseaudio (it seems not to be the preferred way)?

**Comment by JasonLG1979** [comment, marked as answer]
You might have a look at: 

https://github.com/dtcooper/raspotify/wiki/Raspotify-with-PulseAudio-as-a-system-service
